plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseConnectorTest.java

```diff
@@ -91,21 +94,71 @@ public void testColumnName(String columnName)
     public void testRenameColumn()
     {
         // TODO (https://github.com/trinodb/trino/issues/7101) enable the test
+        // ClickHouse need resets all data in a column for specified column which to be renamed
         throw new SkipException("TODO: test not implemented yet");
     }
 
     @Override
     public void testDropColumn()
     {
-        // TODO (https://github.com/trinodb/trino/issues/7101) enable the test
-        throw new SkipException("TODO: test not implemented yet");
+        skipTestUnless(supportsCreateTable());
+
+        String tableName = "test_drop_column_" + randomTableSuffix();
+        // primary key can not be dropped
```

Comment:
I can't see a primary key defined here. Please make this comment more descriptive.

Also it would make more sense to move it where you are testing the DROP COLUMN (i.e. at line 110).

Reply:
I modify it